core/designsystem/src/main/kotlin/com/dhc/designsystem/graph/model/DhcGraphConfig.kt

```diff
@@ -0,0 +1,6 @@
+package com.dhc.designsystem.graph.model
+
+data class DhcGraphConfig(
+    val maxValue: Int = 100,
```

Comment:
```suggestion
    @IntRange(from = 1) val maxValue: Int = 100,
```

```
            Box {
                DhcGraphBar(
                    modifier = Modifier
                        .width(graphWidth)
                        .fillMaxHeight(data[index].value / maxValue.toFloat()),
```
[P2]
maxValue 를 보니 0 이면 zero division error 가 발생할 수 있을 것 같더라구? 
그래서 사소하지만 휴먼에러를 줄이기 위해서 IntrRange Annotation 을 달아주어서 0을 입력할 수 없게 방지해보는것도 나쁘지 않을 것 같아용~